docs/content/reference/migration/migration-0-23.md

```diff
@@ -7,3 +7,47 @@ order: 989
 Previously, you could (confusingly) have two timelines with the same name, as long as they had different types (sequence vs temporal).
 This is no longer possible.
 Timelines are now uniquely identified by name, and if you use different types on the same timeline, you will get a logged warning, and the _latest_ type will be used to interpret the full set of time data.
+
+## 🐍 Python: replaced `rr.set_time_*` with `rr.set_index`
+We're moving towards a more explicit API for setting time, where you need to explicitly specify if a time is either a datetime (e.g. `2025-03-03T14:34:56.123456789`) or a timedelta (e.g. `123s`).
+
+Previously we would infer the user intent at runtime based on the value: if it was large enough, it was interpreted as time since the Unix epoch, otherwise it was interpreted as a timedelta.
+
+To this end, we're deprecated `rr.set_time_seconds`, `rr.set_time_nanos`, as well as `rr.set_time_sequence` and replaced them with `rr.set_index`.
+`set_index` takes either a `sequence=`, `timedelta=` or `datetime=` argument.
+
+`timedelta` must be either:
+* seconds as `int` or `float`
+* `datetime.timedelta`
+* `numpy.timedelta64`.
+
+`datetime` must be either:
+* seconds since unix epoch (1970-01-01) as `int` or `float`
+* `datetime.datetime`
+* `numpy.datetime64`.
```

Comment:
```suggestion
`timedelta` must be either:
* seconds as `int` or `float`
* [`datetime.timedelta`](https://docs.python.org/3/library/datetime.html#datetime.timedelta)
* [`numpy.timedelta64`](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.timedelta64)

`datetime` must be either:
* seconds since unix epoch (1970-01-01) as `int` or `float`
* [`datetime.datetime`](https://docs.python.org/3/library/datetime.html#datetime.datetime)
* [`numpy.datetime64`](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.datetime64)
```